Toggl.Core.UI/ViewModels/Reports/SummaryReportElement.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using Toggl.Core.Reports;
+
+namespace Toggl.Core.UI.ViewModels.Reports
+{
+    public sealed class SummaryReportElement : IReportElement
+    {
+        public float BillablePercentage { get; private set; }
```

Comment:
No need for the private setters if these are set in the constructor
```suggestion
        public float BillablePercentage { get;  }
```